pkg/v1/resource/chartconfig/update.go

```diff
@@ -23,13 +63,50 @@ func (r *Resource) NewUpdatePatch(ctx context.Context, obj, currentState, desire
 		return nil, microerror.Mask(err)
 	}
 
+	delete, err := r.newDeleteChangeForUpdatePatch(ctx, obj, currentState, desiredState)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
 	patch := framework.NewPatch()
 	patch.SetCreateChange(create)
 	patch.SetUpdateChange(update)
+	patch.SetDeleteChange(delete)
 
 	return patch, nil
 }
 
 func (r *Resource) newUpdateChange(ctx context.Context, obj, currentState, desiredState interface{}) ([]*v1alpha1.ChartConfig, error) {
-	return []*v1alpha1.ChartConfig{}, nil
+	currentChartConfigs, err := toChartConfigs(currentState)
```

Comment:
Is the update part right here in this PR? I felt like this got reviewed already and this PR is only about deletion.

Reply:
Yes, this PR implements update. It wasn't there before.